mk_verificator/tests/test_galera_cluster.py

```diff
@@ -5,11 +5,12 @@ def test_galera_cluster_status(local_salt_client):
         ['salt-call mysql.status | grep -A1 wsrep_cluster_size | tail -n1'])
 
     size_cluster = []
+    amount = len(gs)
 
     for item in gs.values():
-        size_cluster.append(item.strip())
+        size_cluster.append(item.split('\n')[-1].strip())
 
-    assert size_cluster == ['3', '3', '3'], \
+    assert size_cluster == ['{}'.format(amount)] * amount, \
```

Comment:
please, use all() statement here. 
http://stackoverflow.com/questions/405516/if-all-in-list-something